install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# By default, this script will install the latest version of run to $HOME/.local/bin.
+install_dir="$HOME/.local/bin"
+version="latest"
+
+# Get the actual version from the latest release
+if [ "$version" = "latest" ]; then
+    version=$(curl -s https://api.github.com/repos/amonks/run/releases/latest | grep tag_name | cut -d '"' -f 4)
+fi
+
+# Detect OS and Architecture
+os=""
+arch=$(uname -m)
+os_type=$(uname -s)
+
+case "$os_type" in
+    Linux*)  os="Linux";;
+    Darwin*) os="Darwin";;
+    *) echo "Unsupported OS" && exit 1;;
+esac
+
+case "$arch" in
+    x86_64*) arch="x86_64";;
+    arm64*)  arch="arm64";;
+    i386*)
+        if [ "$os" = "Linux" ]; then
+            arch="i386"
+        else
+            echo "The i386 architecture is only supported on Linux."
```

Comment:
imo better to try the download and error in one place if it isn't found

that way we can add new (arch) and (uname) values to our release process without also modifying this script